src/pages/iou/request/step/StepScreenWrapper.tsx

```diff
@@ -78,8 +72,4 @@ function StepScreenWrapper({testID, headerTitle, onBackButtonPress, onEntryTrans
     );
 }
 
-StepScreenWrapper.displayName = 'StepScreenWrapper';
-StepScreenWrapper.propTypes = propTypes;
-StepScreenWrapper.defaultProps = defaultProps;
-
-export default StepScreenWrapper;
+export default forwardRef(StepScreenWrapper);
```

Comment:
For some reason, when I was working on that, the steps where throwing errors during render attempt, but it might have been fixed when I changed the callOrReturn content. I already removed it